{{ cookiecutter.__project_name_kebab }}/.pre-commit-config.yaml

```diff
@@ -24,6 +24,11 @@ repos:
       - id: black
         language_version: python3
         args: ['--target-version', 'py37']
+  - repo: https://github.com/asottile/blacken-docs
+    rev: v1.12.1
+    hooks:
+      - id: blacken-docs
+        additional_dependencies: [black==23.9.1]
```

Comment:
I missed bumping this when I upgraded black in PR #52. Also we missed it in review.

Maybe putting black and blacken-docs next to each other will help.

Reply:
could you please also change:
```
repo: https://github.com/adamchainz/blacken-docs
rev: 1.16.0
```

https://github.com/adamchainz/blacken-docs/tags